leveldbjni/src/main/native-package/m4/libtool.m4

```diff
@@ -1,45 +1,37 @@
 # libtool.m4 - Configure libtool for the host system. -*-Autoconf-*-
 #
-#   Copyright (C) 1996, 1997, 1998, 1999, 2000, 2001, 2003, 2004, 2005,
```

Comment:
I'm amazed to see the amount of changes you made to the `libtool.m4`. I'm wondering why those changes are related to our goal?

Reply:
https://github.com/utsaslab/leveldbjni/pull/1/files/4ece7fed84bed249e85ecc72d6f4e81d68a883b9#r219715836